src/main/java/au/com/agiledigital/idea_search/helpers/MacroHelpers.java

```diff
@@ -10,12 +10,15 @@
  * Helpers to do data transformation for macros
  */
 public class MacroHelpers {
+
   public static Set<String> splitTrimToSet(String str, String delimiter) {
```

Comment:
I know this was me who wrote this, but document public function

```suggestion
  /**
   * Splits a string at a delimiter, then remove the starting and ending whitespace from each piece
   * @param str value to be split
   * @param delimiter search key to split the string on
   * @return Set of the sections
   */
  public static Set<String> splitTrimToSet(String str, String delimiter) {
```

Reply:
added